tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationBuilder.cs

```diff
@@ -78,6 +92,28 @@ internal Organisation Build()
                 Address = address,
                 LastUpdated = lastUpdated,
             };
+
+            if (relatedOrganisation != null)
```

Comment:
Code style is to prefer `is not null` to `!= null`.

Probably ought to document some of this stuff and it might seem a tad neurotic again! However, as before keeps things consistent.